config/resolvers/bundleresolver-config.yaml

```diff
@@ -23,6 +23,6 @@ metadata:
     app.kubernetes.io/part-of: tekton-pipelines
 data:
   # the default service account name to use for bundle requests.
-  default-service-account: "default"
+  default-secret: "default-secret"
```

Comment:
Should it be empty instead ? The `default` service account is almost guaranteed (if not 100%) to be in any namespaces. But that's not true for any secret. Default to an empty secret name (aka no secret) would make more sense to me.

Reply:
Yeah that is a good point, I will change it soon

Reply:
fixed